mscore/editstyle.ui

```diff
@@ -247,7 +247,7 @@
        </size>
       </property>
       <property name="currentIndex">
-       <number>4</number>
+       <number>32</number>
```

Comment:
unneeded, but probably harmless

Reply:
For some reason QT 5 Designer preferred number 32 above 4 ;-) Nevertheless, it will be 4 again.

Reply:
Should rather be 0 though

Reply:
Problem is, I cannot test for 0.0 because the "equal" test of real numbers in general is useless. That's why the common practice is to use a range test instead of a equal test.

I don't really understand what you mean with the last sentence. Is this similar as [#305020](https://musescore.org/en/node/305020)?

Reply:
In both 3.x and master the currentIndex is 4. I generally move to the first page after editing one of the tabs but of course I forgot the last time :-(.

Reply:
As mentione that chage is harmless as that index is set programatically anyway.

Reply:
Apparently, since the form always comes up with the right page ;-)

Reply:
first page normally, last active one otherwise, yes

Reply:
Indeed, that's what I called the right page ;-)